src/components/TimeseriesDataExport/TimeseriesDataExport.tsx

```diff
@@ -167,7 +173,13 @@ const TimeseriesDataExportFC: FC<TimeseriesDataExportFormProps> = (
   const fetchTimeseries = async () => {
     const fetchTimeseriesCall =
       retrieveTimeseries || context!.timeseries.retrieve;
-    const timeserieIdsObj = timeseriesIds.map(id => ({ id }));
+
+    const timeserieIdsObj =
```

Comment:
I think @mrtraser suggestion was to merge them together, like it happens on `/timeseries/byids` endpoint

Reply:
True, the main idea to have `timeserieIdsObj: idEither[]` here.

Reply:
Thank you. Hope this time I got everything right :)